<launch>
 <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find velodyne_odo)/cfg/rosconsole.config"/>
	<param name="use_sim_time" value="true"/>
 <node pkg="velodyne_odo" type="vodo_viewer" name="viewer" output="screen" > 
	<!-- launch-prefix="optirun"-->
	<param name="map_frame" value="odom" />
	<!--param name="map_frame" value="base_link" /-->
	<param name="r" value="2.0" />
	<param name="g" value="0.0" />
	<param name="b" value="2.0" />
	<param name="grid_size" value="0.3"/>
	<param name="rad" value="1.0" />
	<param name="max_rad" value="15.0" />
	<param name="tms" value="50" />
	<param name="min_z" value="-10" />
	<param name="max_z" value="10" />
        <remap from="/points" to="/velodyne_odo/normalized_cloud"/>
        <!--remap from="/points" to="/velodyne_points"/-->
 </node>
</launch>
